assets/svg/icons.svg

```diff
@@ -595,5 +595,27 @@
   <symbol id="icon-auction" viewBox="0 0 49 51">
   	<path d="M38.762,51H27.787a1.479,1.479,0,0,1-1.477-1.477V35.449l-12.452.071.071,14A1.478,1.478,0,0,1,12.452,51H1.476A1.478,1.478,0,0,1,0,49.523V22.553a4.964,4.964,0,0,1,1.452-3.517L18.9,6.135a1.84,1.84,0,0,1,1.2-.438,1.955,1.955,0,0,1,1.248.454L24.4,8.4l2.631-3.111a2.872,2.872,0,0,1,.389-.374l.312-.252L27.548,4.3A2.862,2.862,0,0,1,27.9,1.127L28,1a2.817,2.817,0,0,1,1.934-1C30.011,0,30.088,0,30.162,0A2.825,2.825,0,0,1,32,.682l7.342,6.293a2.876,2.876,0,0,1,.327,4.032l-.107.124a2.828,2.828,0,0,1-3.054.854l-.383-.129-.2.353a3.035,3.035,0,0,1-.309.446l-1.429,1.69L48,26.183a2.879,2.879,0,0,1,.322,4.037l-.1.121a2.838,2.838,0,0,1-2.179.976,2.8,2.8,0,0,1-1.823-.654l-3.981-3.412V49.523A1.478,1.478,0,0,1,38.762,51Zm-24.9-17.028H26.381a1.479,1.479,0,0,1,1.477,1.477V49.523l10.9-.071L38.7,25.933l-8.293-7.112-1.433,1.7a2.866,2.866,0,0,1-.572.5l-.368.253.213.39a2.9,2.9,0,0,1-.316,3.233l-.1.121a2.83,2.83,0,0,1-2.177.977,2.79,2.79,0,0,1-1.823-.655l-7.34-6.29a2.882,2.882,0,0,1-.326-4.036l.105-.122a2.834,2.834,0,0,1,3.109-.828l.422.151.191-.409a2.857,2.857,0,0,1,.394-.647L23.4,9.583l-3.03-2.237a.382.382,0,0,0-.5,0L2.459,20.2a3.431,3.431,0,0,0-.911,2.348v26.97l10.9-.071-.071-14A1.478,1.478,0,0,1,13.857,33.972ZM33.089,15.647l-1.582,1.872,13.81,11.839a1.162,1.162,0,0,0,.743.273c.03,0,.06,0,.09,0a1.151,1.151,0,0,0,.781-.4l.1-.122a1.155,1.155,0,0,0-.13-1.618L33.088,15.647Zm-14.649-.036a1.16,1.16,0,0,0-.881.385l-.1.122a1.159,1.159,0,0,0,.13,1.618l7.34,6.291a1.126,1.126,0,0,0,.736.27c.032,0,.065,0,.1,0a1.139,1.139,0,0,0,.782-.4l.1-.122a1.154,1.154,0,0,0-.13-1.615l-7.34-6.291A1.121,1.121,0,0,0,18.439,15.611ZM29.183,5.987l-.241.054a1.11,1.11,0,0,0-.617.361l-6.647,7.863a1.166,1.166,0,0,0,.132,1.616l4.257,3.65a1.13,1.13,0,0,0,.738.273c.032,0,.064,0,.1,0a1.143,1.143,0,0,0,.781-.4l6.649-7.861a1.152,1.152,0,0,0,.263-.657l.015-.246Zm.988-4.257a1.156,1.156,0,0,0-.879.387l-.1.122a1.152,1.152,0,0,0,.13,1.616l7.34,6.292a1.132,1.132,0,0,0,.736.27c.032,0,.065,0,.1,0a1.138,1.138,0,0,0,.78-.4l.1-.122a1.155,1.155,0,0,0-.13-1.616l-7.34-6.292A1.131,1.131,0,0,0,30.172,1.729Z"/>
   </symbol>
+
+  <symbol id="icon-home" viewBox="0 0 274 274">
```

Comment:
<img width="262" alt="Screenshot 2022-10-24 at 11 29 48" src="https://user-images.githubusercontent.com/9892048/197506318-33407048-b62b-4b86-881e-5c370fb96920.png">

Is this what the icon should look like?

Reply:
Yes

Reply:
Are you referring to the line on the left?

It looks like this in the design system:

![image](https://user-images.githubusercontent.com/74235311/197507532-4a13cdcb-0174-4415-a80d-fcd53ce8fa27.png)